saleor/channel/migrations/0001_initial.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 3.0.6 on 2020-06-16 07:54
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+def get_default_currency(Checkout, Order, Product, ShippingMethod, Voucher):
+    latest_product = Product.objects.order_by("-pk").first()
+    if latest_product:
+        return latest_product.currency
+    latest_voucher = Voucher.objects.order_by("-pk").first()
+    if latest_voucher:
+        return latest_voucher.currency
+    latest_shipping_method = ShippingMethod.objects.order_by("-pk").first()
+    if latest_shipping_method:
+        return latest_shipping_method.currency
+    latest_order = Order.objects.order_by("-pk").first()
+    if latest_order:
+        return latest_order.currency
+    latest_checkout = Checkout.objects.order_by("-pk").first()
+    if latest_checkout:
+        return latest_checkout.currency
+    return None
+
+
+def create_default_channel(apps, schema_editor):
+    Channel = apps.get_model("channel", "Channel")
+    Checkout = apps.get_model("checkout", "Checkout")
+    Order = apps.get_model("order", "Order")
+    Product = apps.get_model("product", "Product")
+    ShippingMethod = apps.get_model("shipping", "ShippingMethod")
+    Voucher = apps.get_model("discount", "Voucher")
+
+    default_currency = get_default_currency(
+        Checkout, Order, Product, ShippingMethod, Voucher
+    )
+    if default_currency:
+        Channel.objects.create(
+            name="Default channel",
```

Comment:
Please make sure that default channel is active. In case of migrating existing environment to multichannel all products are going to be assigned to inactive default channel.